spec/features/web_helper.rb

```diff
@@ -13,3 +14,29 @@ def unsuccessful_sign_up_with_wrong_password
   fill_in 'Last name', with: "test_last_name"
   fill_in 'Email', with: "test_email@keepingitrails.com"
 end
+
+
+#sign in
+def successful_sign_in
+  visit "/"
```

Comment:
these two lines repeat 3 times, could extract these to a separate method called click_already_user